Tracing Demo DRAFT (#11518)
getsentry/sentry-docs: docs/product/sentry-basics/distributed-tracing/generate-first-error.mdx

        Nonfat Water
       </button>
    </div>
```
Take note that in the `server.js` file of the `tracing-tutorial-backend` repo, there is a middleware function handling the `/products/debug-sentry` route, which throws a sample error:

```javascript {filename:server.js}
app.get("/products/debug-sentry", (req, res) => {
    console.log('Sentry Error thrown!');
    throw new Error("My first Sentry error!");
  });
```

1. Save the file.

1. Go back to the browser window and try clicking on the **Nonfat Water** button again, you will no longer see the expected product information. Instead, you'll see text saying something went wrong.  If you look at line 88 of the `App.js` file in the `tracing-tutorial-frontend` repo, you'll see this is what the frontend displays when it's call to the backend doesn't return any data.

1. Open the terminal window that's running the `tracing-tutorial-backend` server. Here, you'll see that a log letting you know that an error has occurred.

```bash
Sentry Error thrown!
```

## 2. View the Error in Sentry

Now that you've triggered an error, let's see what it looks like in Sentry.

> 1. Open the **Traces** page in [Sentry.io](https://sentry.io/). Make sure one (or both) of the projects you are using here is selected in the projects dropdown.
> 

Reviewer: ```suggestion
1. Open the [**Traces**](https://sentry.io/orgredirect/organizations/:orgslug/traces/) page in Sentry.io. Make sure one (or both) of the projects you're using for this tutorial is selected in the projects dropdown.

```

2. You should see a trace that shows both the React and Express icons, indicating that this trace traverses both projects. Click on the blue trace ID on the left to see the Trace View.

> 3. On the Trace View page, you can see every span that happened within this trace. You can see the error on the React app that directly triggered the error on the Express app.

Reviewer: ```suggestion
3. On the [Trace View](/concepts/key-terms/tracing/trace-view/) page, you can see every span that happened within this trace. You can see the error on the React app that directly triggered the error on the Express app.
```

### UI Walkthrough

The interactive demo below walks through how to view a distributed trace in Sentry.

<div style={{"height":"0px","paddingBottom":"calc(52.1021021021021% + 41px)","position":"relative","width":"100%"}}>
  <iframe
    src="https://demo.arcade.software/xPc1a1WUYTnQqPvBQGnQ?embed"
    frameborder="0"
    loading="lazy"
    webkitallowfullscreen
    mozallowfullscreen
    allowfullscreen
    allow="fullscreen;"
    style={{"colorScheme":"light","height":"100%","left":"0px","position":"absolute","top":"0px","width":"100%"}}
    title="FE Tutorial - View First Error"
  ></iframe>
</div>

## Next

> At this point, you've verified that Sentry is providing distributed tracing across your projects. You can now dig in to issues to easily find the original source of the issue, so you can diagnose and fix it in less time.

Reviewer: ```suggestion
Congrats, you're done! You've verified that Sentry is providing distributed tracing across your projects. You can now dig in to issues to find the original source of the issue that's causing a problem so you can diagnose and fix it in less time.
```
